docs/flyte_agents/using_agents_in_tasks.md

```diff
@@ -0,0 +1,14 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+(using_agents_in_tasks)=
+# Using agents in tasks
+
+If you need to connect to an external service in your workflow, we recommend using the corresponding agent rather than a web API plugin. Agents are designed to be scalable and can handle large workloads efficiently, and decrease load on FlytePropeller, since they run outside of it. You can also test agents locally without having to change the Flyte backend configuration, streamlining development.
+
+For a list of agents you can use in your tasks, see the [Agent Setup](https://docs.flyte.org/en/latest/deployment/agents/index.html) documentation.
```

Comment:
Here it would be better to link to the Integrations section of the docs, specifically the [External Service Backend Plugins section](https://docs.flyte.org/en/latest/flytesnacks/integrations.html#external-service-backend-plugins), but that section contains plugins that don't have a corresponding agent (e.g. AWS Athena). Should I add a new "Agents" section to that Integrations page?

Reply:
Sounds good